### DISCUSSION TITLE
Art

### DISCUSSION THREAD

**Discussion by nimrossum** [git-truck/git-truck#79] [Fun and games]
This thread is for posting artistic screenshots of the app.

**Comment by nimrossum** [comment]
*Clusterfuck*
![image](https://user-images.githubusercontent.com/1959615/154696408-cade7245-d752-4044-8b7e-86fbbca168a3.png)
From visualizing [zeeguu frontend](https://github.com/zeeguu-ecosystem/zeeguu-react)

**Comment by nimrossum** [comment]
*Hollow circles*
![localhost_3000_](https://user-images.githubusercontent.com/1959615/154696086-0b48a37b-b40c-4060-8360-f21920221bbc.png)

**Comment by nimrossum** [comment]
*Pale black dot*
![localhost_3000_ (1)](https://user-images.githubusercontent.com/1959615/154699919-87584de0-7023-4a1c-ba77-6fd737072053.png)
From when radius went `NaN`

**Comment by emiljapelt** [comment]
![billede](https://user-images.githubusercontent.com/55390848/154711846-217abd1f-c997-47bc-aa54-9c2fcf00e9df.png)
First file extension base coloring.

**Comment by emiljapelt** [comment]
![billede](https://user-images.githubusercontent.com/55390848/154712088-e3fd3773-4b5c-4c4d-947f-3db8329bd2ff.png)
First heatmap coloring

**Comment by emiljapelt** [comment]
![billede](https://user-images.githubusercontent.com/55390848/154712241-d8460e3f-c487-4071-8b57-b7409d3b3c29.png)
First coldmap coloring.

**Comment by nimrossum** [comment]
*Tarteletfyld*
![273803860_490750572432426_3300028848653579588_n](https://user-images.githubusercontent.com/1959615/154713933-d9dd47db-0d3a-4ee4-81fc-7490b21015a4.png)
No size mapping applied

**Comment by nimrossum** [comment]
*M'tp dign tcp*
![localhost_3000_ (2)](https://user-images.githubusercontent.com/1959615/154724639-19ef903f-08ff-4b80-9e26-b4c2c439a6b8.png)

**Comment by tjomson** [comment]
Fixed muted colors for visualization (there was a 0.3 opacity on them)
HeatMap
![billede](https://user-images.githubusercontent.com/23435481/154997113-3b3691c8-db56-437e-b1f5-61d42e46bbe8.png)

File extensions:
![billede](https://user-images.githubusercontent.com/23435481/154997166-f99e9558-fcdd-4d76-9d00-32d4fca57e94.png)

**Comment by nimrossum** [comment]
@mircealungu **First Treemap**
![image](https://user-images.githubusercontent.com/1959615/155219427-e7aae870-3e5e-4470-aac1-2998e72ed1cc.png)

**Comment by emiljapelt** [comment]
The blob is taking over
![billede](https://user-images.githubusercontent.com/55390848/155697301-1f9974ba-d772-4541-bc6e-9240d4200beb.png)
Radius increase

**Comment by tjomson** [comment]
![billede](https://user-images.githubusercontent.com/23435481/155736142-3ba0647f-7ca9-4c31-9973-8d8bda213c03.png)

**Comment by tjomson** [comment]
chunker compenhagen.bin
![billede](https://user-images.githubusercontent.com/23435481/155983747-e70d88bd-73bd-48b2-9010-416ba4c724fe.png)

**Comment by hojelse** [comment]
![art](https://user-images.githubusercontent.com/34659757/157014078-142f75db-4864-41e0-a749-76c92a1c65ed.png)

**Comment by hojelse** [comment]
![art](https://user-images.githubusercontent.com/34659757/157014157-ecd3d9cf-8a1f-4892-bf8b-2e92e41a00cf.png)

**Comment by hojelse** [comment]
![fluff](https://user-images.githubusercontent.com/34659757/157014352-ea9a1b93-a565-40cb-9e31-9dccc2da5505.png)

**Comment by nimrossum** [comment]
*The patient seems to show signs of personality disorder*


![image](https://user-images.githubusercontent.com/1959615/157863247-344b3b14-7624-436a-b57e-6c04f5377eaa.png)

**Comment by nimrossum** [comment]
*Cursed little devil*
![localhost_3000_ (11)](https://user-images.githubusercontent.com/1959615/158151059-0fce9763-332a-44ac-b981-3f1ef430754e.png)

**Comment by nimrossum** [comment]
*It looks like hair?*
![image](https://user-images.githubusercontent.com/1959615/158408595-a690f673-4649-4193-9e84-f5010d379dac.png)

**Comment by mircealungu** [comment]
package-lock and yarn-lock welcoming me as I open a student project: 

<img width="1662" alt="image" src="https://user-images.githubusercontent.com/464519/158686718-427ac79f-51ec-4b64-a121-58fcdea0be9f.png">

**Reply by nimrossum** [reply]
This might tell us that they are inconsistent with the use of yarn and npm? 

In other "news", yarn can create a yarn.lock file from package-lock.json

https://classic.yarnpkg.com/blog/2018/06/04/yarn-import-package-lock/

**Comment by tjomson** [comment]
Part of TensorFlow
![billede](https://user-images.githubusercontent.com/23435481/165912105-f43d0c59-e48b-45e2-9567-57f40afe8f0d.png)

**Reply by mircealungu** [reply]
How can they have so many files in the same folder?!

**Comment by mircealungu** [comment]
Beautiful picture and wonderful idea to look at a highly collaborative project :)

**Comment by emiljapelt** [comment]
Orange halfmoon
![Udklip](https://user-images.githubusercontent.com/55390848/165941121-dce05c39-0611-4c5f-a37b-7264f7b9bbbe.PNG)

**Reply by mircealungu** [reply]
This made me think about one possible way to increase the "virality" of git-🚚  could be to create a cozy place (reddit thread? another GH  conversation here?) where people can share the visualization of their system?

**Reply by tjomson** [reply]
![IMG_20220429_145437](https://user-images.githubusercontent.com/23435481/165948270-22249cf7-e2d9-4db4-9be9-2ebbe6f84b72.jpg)
I am starting the truck-fie trend

**Comment by nimrossum** [comment]
![image](https://user-images.githubusercontent.com/1959615/166251122-0f1bf8b9-ecbd-4df2-802e-248dbe66e2ac.png)

**Reply by mircealungu** [reply]
removing the lines reminds me of the principle from Tufte about "maximizing data-ink" (https://infovis-wiki.net/wiki/Data-Ink_Ratio)

**Comment by mircealungu** [comment]
## Typescript, or the importance of tests when you write a compiler :) 

![image](https://user-images.githubusercontent.com/464519/167704411-8b82582e-fc01-4d67-a84f-9a9d0bf7fa67.png)

Also, this is incredibly slow to render :)

**Reply by mircealungu** [reply]
## File Types
![image](https://user-images.githubusercontent.com/464519/167705071-cfc76f26-d878-43ce-8524-0fecb1a5f43a.png)

## Solitary authors only in the tests :)

![image](https://user-images.githubusercontent.com/464519/167705274-69779fb4-47cd-43da-88e6-fbca137dfc1d.png)

**Reply by nimrossum** [reply]
How long did it take to analyze? And how long to load visualization?

**Reply by mircealungu** [reply]
✔ Found branch head [15ms]
✔ Commit tree analyzed [1.59s]
✔ Commit tree hydrated [144.47s]
✖ Error writing data to file /usr/local/lib/node_modules/git-truck/.temp/TypeScript/main.json
GET /TypeScript/main?_data=routes%2F%24repo.%24 200 - - 147685.789 ms

**Reply by tjomson** [reply]
> ## Typescript, or the importance of tests when you write a compiler :)
> 
> ![image](https://user-images.githubusercontent.com/464519/167704411-8b82582e-fc01-4d67-a84f-9a9d0bf7fa67.png)
> 
> Also, this is incredibly slow to render :)

Very cool fading effect :o

We have made a tree map with less padding, which you can try out using git-truck@testing if you want.

I don't think there is a whole lot to do about the performance. The advantage of the tree map is that more items can be shown, but that also means that a lot of items have to be rendered :/

**Reply by nimrossum** [reply]
This is now fixed using @tjomson 's awesome PR #698 🥳

**Comment by nimrossum** [comment]
*Green stripes*

![image](https://user-images.githubusercontent.com/1959615/168447800-cfbc3f51-f635-4a13-9077-9df0fafd65d3.png)

**Comment by nimrossum** [comment]
*Partition layout*

![image](https://user-images.githubusercontent.com/1959615/168448011-666f1e80-6fc9-4462-b6d7-e49d4491f7b0.png)

**Comment by nimrossum** [comment]
*Fried egg*
![image](https://user-images.githubusercontent.com/1959615/168448725-b97b90d5-63fc-4c26-ab9b-28e6519084a2.png)

**Comment by nimrossum** [comment]
![image](https://user-images.githubusercontent.com/1959615/168448908-8225e0d8-71ce-4a90-9dba-b3fb5ad06f10.png)

**Comment by nimrossum** [comment]
Word cloud tree map
![image](https://github.com/git-truck/git-truck/assets/1959615/cb0cf35f-7bed-4898-bb16-7384943ebc6e)

**Comment by hojelse** [comment]
linux
<img width="1761" alt="git-truck-r3f-2" src="https://github.com/git-truck/git-truck/assets/34659757/e966a030-9e5f-4e4a-8278-9ad671fec26f">

**Reply by mircealungu** [reply]
obviously!

**Comment by tjomson** [comment]
![received_860848642189328](https://github.com/git-truck/git-truck/assets/23435481/746dd978-0580-45f4-b83c-1251afc54d3a)

**Comment by nimrossum** [comment]
[Vite](https://github.com/vitejs/vite/)

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/83827c62-6dcb-4e1a-bfed-34762f7ae7d4" />

**Comment by nimrossum** [comment]
Git
<img width="816" height="728" alt="image" src="https://github.com/user-attachments/assets/a0ddd13a-5237-4541-94b6-cc2e62464a11" />

**Comment by nimrossum** [comment]
[V8](https://github.com/v8/v8)

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/af3ec089-7ec4-40a2-9da2-c69671210b85" />

**Comment by gustavmueller** [comment]
<img width="1280" height="701" alt="image" src="https://github.com/user-attachments/assets/19c5109a-74ab-4159-a5b1-4a391a25da1d" />

**Reply by mircealungu** [reply]
Whoa! That's ART!

**Comment by gustavmueller** [comment]
Nr #1 repo
<img width="950" height="869" alt="image" src="https://github.com/user-attachments/assets/9b959774-ca4b-4fb7-afb9-1bc9d6a1211b" />

**Reply by gustavmueller** [reply]
As SVG export! 
<img width="950" height="869" alt="image" src="https://github.com/user-attachments/assets/7110c960-74c6-4d43-b5fe-9790632b86c0" />
